### PR TITLE
Bug 1400015 - optimize environment for running merge day/staging rele…

### PR DESCRIPTION
**Pull request by kmoir**
…ase scripts

### CODE REVIEW

File: how-tos/merge-and-staging-instance.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Staging and merging instance
+
+There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos.  
+
+To access it, you can either 
+1. Create an new instance with the AMI named "Mozilla release staging/merge machine".  I used a M3.large instance with 32 gb space added.
```
Comment (MihaiTabara) [inline how-tos/merge-and-staging-instance.md]: formatting tiny nit: can you please enclose the machine type within code-quotes like ```this```? I find it easier to spot and copy-paste info if it's highlighted.

File: how-tos/merge-and-staging-instance.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+2. Use the existing instance named "Mozilla release staging/merge machine"
+You will need a pem file to access them, the key for it is the usual location
```
Comment (MihaiTabara) [inline how-tos/merge-and-staging-instance.md]: Might be worth adding more people's keys to it. So far I only see you, Johan and me but we should have at least all the people on releaseduty hook for 57.

File: how-tos/setup-staging-release.md [hunk 3]
```diff
@@ -21,12 +21,14 @@ Use cases:
 
 ## Merge scripts
 
-1. Use the <a href="../scripts/staging_merge.py">script</a>  to run the merge for you. Parameters are the directory to run the merge in, and the repos to merge_from, and merge_to.  For example:
+1.  See the doc <a href="merge-and-staging-instance.md">aws staging instance</a> to see if using a dedicated instance for staging releases would be helpful.
```
Comment (MihaiTabara) [inline how-tos/setup-staging-release.md]: formatting tiny nit: s/aws/AWS for consistency.

**Review by MihaiTabara — APPROVED** [review]

**Comment by kmoir** [comment]
I'll add the other releng users to the encrypted keys.  I had problems adding them on Friday but will investigate more.

**Comment by kmoir** [comment]
Okay keys are now there.

**Comment by MihaiTabara** [comment]
Thanks!

**Pull request merged**